agent/agent.go

```diff
@@ -62,6 +62,7 @@ var (
 	debug                = flag.Bool("debug", false, "Whether or not to print debug log messages")
 	disableSSLForTest    = flag.Bool("disable-ssl-for-test", false, "Disable requirements for SSL when running tests locally")
 	favIconURL           = flag.String("favicon-url", "", "URL of the favicon")
+	testServiceIdentity  = flag.String("test-service-identity", "", "Service identity, that will be used by the agent, when testing locally")
```

Comment:
We should refer to these as "VM identity" instead of "Service identity", both in the flag/variable names and in the text description.